### PR TITLE
RTL fix, add Arabic translation and config fix

### PR DESCRIPTION
**Pull request by mrkindy**
RTL fix, add Arabic translation and config fix

### CODE REVIEW

File: resources/views/master.blade.php [hunk 1]
```diff
@@ -37,6 +37,11 @@
         .voyager .breadcrumb a{
             color:{{ config('voyager.primary_color','#22A7F0') }};
         }
+        @if(config('voyager.multilingual.rtl'))
```
Comment (emptynick) [inline resources/views/master.blade.php]: This is not necessary anymore. It was taken care of in https://github.com/the-control-group/voyager/pull/3754

**Review by emptynick — DISMISSED** [review]

File: resources/views/master.blade.php (hunk 1 above)
Comment (mrkindy) [reply resources/views/master.blade.php]: Done

**Comment by mrkindy** [comment]
Done

**Comment by emptynick** [comment]
I think this is fine then?
It will fall-back to the given language-string if it doesn't exist anyway.

File: publishable/config/voyager.php [hunk 2]
```diff
@@ -132,17 +132,17 @@
     'dashboard' => [
         // Add custom list items to navbar's dropdown
         'navbar_items' => [
-            'Profile' => [
+            'voyager::generic.profile' => [
```
Comment (emptynick) [inline publishable/config/voyager.php:135]: The same changes you made to this file also need to be done in `voyager_dummy.php`

**Review by emptynick — CHANGES_REQUESTED** [review]

File: publishable/config/voyager.php (hunk 2 above)
Comment (mrkindy) [reply publishable/config/voyager.php:135]: Done

**Review by emptynick — APPROVED** [review]

**Comment by emptynick** [comment]
Documented in https://github.com/the-control-group/voyager/pull/3909/commits/aba010917cd0d2e037db5e0f97e0c40415420744

**Pull request merged**